dev-infrastructure/Makefile

```diff
@@ -376,3 +377,10 @@ clean: svc.clean mgmt.clean region.clean
 
 local-cs-permissions: svc.dev.permissions mgmt.dev.permissions
 .PHONY: local-cs-permissions
+
+#
+# Generated Prometheus Rules
+#
+modules/metrics/rules/generatedPrometheusAlertingRules.bicep:
```

Comment:
@geoberle is there some better place to put this generation step to make sure it stays in sync?

Reply:
do i understand the idea correctly that you would like to regenerate the bicep file when the alert yaml changes? and then detect a drift in the context of `fmt` and `lint` ?

Reply:
Yeah, if someone updates the source files I would like a CI check that tells them they need to run this generation step before merge so that the output stays in sync with the input.

Reply:
the bicep-lint.yaml gh action already fails the PR if uncommited changes

Reply:
@geoberle awesome, added this to that check. should be good to go now